browser-app/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "private": true,
+  "name": "browser-app",
+  "version": "0.0.1",
+  "dependencies": {
+    "@theia/core": "latest",
```

Comment:
Set up, please, concrete version of dependencies 0.3.8 for example.

Reply:
done